keywords/TestServeriOS.py

```diff
@@ -174,8 +174,11 @@ def install(self):
 
         log_info("Installing: {}".format(self.app_path))
         # Launch the simulator and install the app
+        # output = subprocess.check_output([
+        #     "ios-sim", "--devicetypeid", self.device, "install", self.app_path, "--exit"
+        # ])
         output = subprocess.check_output([
-            "ios-sim", "--devicetypeid", self.device, "install", self.app_path, "--exit"
+             "ios-sim", "install", self.app_path, "--devicetypeid", self.device, "--exit"
```

Comment:
Looks like there is no change here

Reply:
There is a change. seeing issues to get device id when ios-sim installed by brew, so did yum install and that needs this order change, otherwise it won't work